src/diva-react/services/diva-service.js

```diff
@@ -6,6 +6,7 @@ function startIrmaSession(irmaSessionType, options) {
     .post('/api/start-irma-session', {
       type: irmaSessionType,
       content: options.attributesRequired,
+      message: options.message, // if options.message is undefined, field won't be included in body
```

Comment:
the same applies to content and credentialType, so, I don't think this comment is useful here, or it should be generalized to something like 'only properties that are passed to the service will actually be included in the request).